File: KarenNgala/minutes_to_seconds.py
# function
def convert(min):
    return min*60

# test value 
print("5 mins: " + str(convert(5)) + " seconds")
print("3 mins: " + str(convert(3)) + " seconds")


# prompt user to enter value
minutes = input("Enter minutes to convert to seconds: ")

print(convert(int(minutes)))

File: prince475/minutesToseconds_conversion.js/minutesToseconds.java
import java.util.Scanner;
class minutesToSeconds
{
    public static void main(String[] arg)
    {
        int min,sec;
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter Minutes");
        min=sc.nextInt();
        sec=minToSec(min);
        System.out.println("Seconds: "+sec);
    }
    static int minToSec(int x)
    {
        return x*60;

    }
}